save-backend/src/main/kotlin/com/saveourtool/save/backend/controllers/TestSuitesSourceController.kt

```diff
@@ -0,0 +1,262 @@
+package com.saveourtool.save.backend.controllers
+
+import com.saveourtool.save.backend.ByteBufferFluxResponse
+import com.saveourtool.save.backend.service.*
+import com.saveourtool.save.backend.storage.TestSuitesSourceSnapshotStorage
+import com.saveourtool.save.entities.TestSuite
+import com.saveourtool.save.entities.TestSuitesSource
+import com.saveourtool.save.testsuite.*
+import com.saveourtool.save.utils.*
+import com.saveourtool.save.v1
+import org.slf4j.Logger
+import org.springframework.http.HttpStatus
+import org.springframework.http.MediaType
+import org.springframework.http.ResponseEntity
+import org.springframework.http.codec.multipart.Part
+import org.springframework.web.bind.annotation.*
+import reactor.core.publisher.Mono
+import reactor.kotlin.core.publisher.toFlux
+import reactor.kotlin.core.publisher.toMono
+
+typealias TestSuiteList = List<TestSuite>
+
+/**
+ * Controller for [TestSuitesSource]
+ */
+@RestController
+@RequestMapping("/internal/test-suites-sources", "/api/$v1/test-suites-sources")
+class TestSuitesSourceController(
```

Comment:
Could you please write all the comments in swagger style

Reply:
added